lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -51,16 +53,21 @@ public DocumentCheckVerificationResult verifyIdentity(DrivingPermitForm drivingP
         DocumentCheckVerificationResult result = new DocumentCheckVerificationResult();
 
         try {
-            LOGGER.info("Validating identity...");
+            LOGGER.info("Validating form data...");
             ValidationResult<List<String>> validationResult =
-                    this.personIdentityValidator.validate(drivingPermitData);
+                    this.formDataValidator.validate(drivingPermitData);
             if (!validationResult.isValid()) {
-                result.setExecutedSuccessfully(false);
-                result.setValidationErrors(validationResult.getError());
-                result.setError("IdentityValidationError");
-                return result;
+                String errorMessages = String.join(",", validationResult.getError());
+                LOGGER.error(
+                        "{} - {} ",
+                        ErrorResponse.FORM_DATA_FAILED_VALIDATION.getMessage(),
```

Comment:
I don't think this will return PII however worth verifying against real DCS validation error to be sure...If memory serves it returns the field that is failing validation without the value so should be fine